components/header.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+export default function Header() {
+  return (
+    <>
+      <Headerbody>
+        <AppHeadline>MediHelp</AppHeadline>
+        <AbcdeLink>ABCDE</AbcdeLink>
+      </Headerbody>
+    </>
+  );
+}
+
+const Headerbody = styled.header`
+  background-color: hotpink;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+`;
+
+const AppHeadline = styled.p``;
+
+const AbcdeLink = styled.p``;
```

Comment:
if a styled component has no CSS properties, you can just use normal html tags instead

Reply:
This is a buildup for later.

Reply:
I see. Then I would leave it for the moment.

But for future, you should only create components that you actually use in the corresponding user story :)